src/main/java/io/split/android/client/utils/Utils.java

```diff
@@ -84,4 +84,20 @@ public static boolean isReachable(String host, int port, int timeoutMs ) {
             return true;
         } catch (IOException e) { return false; }
     }
+
+    public static String sanitizeForFileName(String string) {
+        if(string == null) {
+            return "";
+        }
+        return string.replaceAll("[^a-zA-Z0-9\\.\\-]", "_");
+    }
+
+    public static String join(String separator, Iterable<String> iterable) {
+        StringBuilder builder = new StringBuilder();
```

Comment:
dont you have Strings.join or some guava utility?

Reply:
I've removed this function in previous PR, so  it will be deleted when merging.